azkaban-exec-server/src/main/java/azkaban/execapp/event/BlockingStatus.java

```diff
@@ -13,14 +13,14 @@
  * License for the specific language governing permissions and limitations under
  * the License.
  */
-
 package azkaban.execapp.event;
 
 import azkaban.executor.Status;
 
+
 public class BlockingStatus {
 
-  private static final long WAIT_TIME = 5 * 60 * 1000;
+  private static final long WAIT_TIME = 300000;  // 5 * 60 * 1000
```

Comment:
or Duration ?

Reply:
what do you mean? I didn't intend to change variable name. Just don't want to confuse the compiler to do extra steps.